docs/batch-changes/configuring-credentials.mdx

```diff
@@ -187,3 +189,43 @@ Follow the steps to [create a PAT](https://learn.microsoft.com/en-us/azure/devop
 When Sourcegraph is configured to [clone repositories using SSH via the `gitURLType` setting](/admin/repo/auth), an SSH keypair will be generated for you, and the public key needs to be added to the code host to allow push access. You will be given that public key when adding your personal access token. You can also come back later and copy and paste it into your code host's SSH access settings page.
 
 ![create-credential-ssh-key](https://sourcegraphstatic.com/docs/images/batch_changes/create-credential-ssh-key.png)
+
+## GitHub Apps
+
+This is only available for GitHub code hosts.
+
+GitHub apps follow the same concepts as [personal and global access tokens](#types-of-access-tokens-used-by-batch-changes).
+
+### Adding a personal GitHub app
```

Comment:
We can drop the `personal` word here. No need for it here.
It can lead to confusion, we already stated earlier that there's two types of credentials:

1. Personal Credentials
2. Global service credentials

@bahrmichael

Reply:
I pushed an update. Let me know what you think.